gbpservice/nfp/configurator/drivers/firewall/vyos/vyos_fw_driver.py

```diff
@@ -121,6 +189,26 @@ def configure_interfaces(self, context, resource_data):
 
         """
 
+        mgmt_ip = resource_data['mgmt_ip']
```

Comment:
Configure health monitoring is the first API that lands on the drivers. Shouldn't we do the log forwarding once the health check works?